app/scripts/background.js

```diff
@@ -1596,23 +1596,34 @@ async function triggerUi() {
 }
 
 // It adds the "App Installed" event into a queue of events, which will be tracked only after a user opts into metrics.
-const addAppInstalledEvent = () => {
+const addAppInstalledEvent = async () => {
   if (controller) {
     controller.metaMetricsController.updateTraits({
       [MetaMetricsUserTrait.InstallDateExt]: new Date()
         .toISOString()
         .split('T')[0], // yyyy-mm-dd
     });
+
+    const deferredDeepLink =
```

Comment:
a set method shouldn't get a value. I think you should call `getDeferredDeepLinkFromCookie` here, and pass it to `setDeferredDeepLink`.